spi/src/main/java/org/opensearch/jobscheduler/spi/utils/SchemaUtils.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.jobscheduler.spi.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.ActionListener;
+import org.opensearch.action.admin.indices.mapping.put.PutMappingRequest;
+import org.opensearch.action.support.master.AcknowledgedResponse;
+import org.opensearch.client.IndicesAdminClient;
+import org.opensearch.cluster.ClusterState;
+import org.opensearch.cluster.metadata.IndexMetadata;
+import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.common.xcontent.LoggingDeprecationHandler;
+import org.opensearch.common.xcontent.NamedXContentRegistry;
+import org.opensearch.common.xcontent.XContentParser;
+import org.opensearch.common.xcontent.XContentParser.Token;
+import org.opensearch.common.xcontent.XContentType;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SchemaUtils {
+    private static final String _META = "_meta";
+    private static final String SCHEMA_VERSION = "schema_version";
+    private static final String _DOC = "_doc";
+    private static final String LOCK_INDEX_NAME = ".opendistro-job-scheduler-lock";
```

Comment:
is it not defined already in the package? if it is can you reuse

Reply:
I don't think these are defined anywhere else in the package.

Reply:
https://github.com/opensearch-project/job-scheduler/blob/41a38baec560d4a085412366d227836e9ede4a7f/spi/src/main/java/org/opensearch/jobscheduler/spi/utils/LockService.java#L70 it looks like the index name is defined here too - can you define it at only place and reuse at other?